db/migrate/20230726144101_add_confirmable_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddConfirmableToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :confirmation_token, :string
+    add_column :users, :confirmed_at, :datetime
+    add_column :users, :confirmation_sent_at, :datetime
+    add_column :users, :unconfirmed_email, :string
+    add_index :users, :confirmation_token, unique: true
+  end
+end
```

Comment:
:red_circle: [REQUIRED] I notice that this migration update the users' table to add the confirmable columns. However, you've also uncommented the confirmable columns in your `add_devise_to_users` migration. This means that you're already using the Devise confirmable module, which means that these columns will be added automatically.

As a result, there's no need to add the code above to your migration. Doing so will actually lead to errors when running `rails db:migrate.`

To avoid this, please make sure to either:
- Delete the AddConfirmableToUsers migration file.
- Comment out the confirmable columns in your add_devise_to_users migration.

Reply:
Done, aa advised ,  I had already deleted the migration, however , had not pushed it